pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.ruff]
+# Ignore some well known paths
+exclude = [
+  ".tox",
+  ".venv",
+  "*.egg",
+  "build",
+  "db/env.py",
+  "db/versions/*.py",
+  "dist",
+  "doc",
+  "site"
+]
+format = "github"
```

Comment:
...and here the github format is specified as well.

Reply:
Removed.